docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+letsencrypt:
+  build: .
+  ports:
+    - "443:443"
+  volumes:
+    - /etc/letsencrypt:/etc/letsencrypt/certs
```

Comment:
Ups, this was meant to be `/etc/letsencrypt:/etc/letsencrypt`, thanks for catch!